__tests__/db.test.js

```diff
@@ -219,10 +219,40 @@ describe('App.js', () => {
           .expect(200)
           .then((res) => {
             expect(res.body.length).toEqual(4)
-            expect(res.body[0]).toMatchObject({
-              username: 'butter_bridge',
-              name: 'jonny',
-              avatar_url: 'https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg'
+            res.body.forEach((user) => {
+              expect(typeof user.username).toBe('string')
+              expect(typeof user.name).toBe('string')
+              expect(typeof user.avatar_url).toBe('string')
+            })
+          })
+        })
+        test('GET - /api/articles?:topic - should return status code 200 and a list of all filtered articles by specified topic and still remains in descending order by created_at', () => {
+          return request(app)
+          .get('/api/articles?topic=cats')
+          .expect(200)
+          .then((res) => {
```

Comment:
It's essential to check before a beforeEach for the length of the response, otherwise if the response comes back empty for any reason the forEach won't run meaning the assertions won't run and the test will auto pass unexpectedly.